### DISCUSSION TITLE
Underestimating SALT due to lacking property tax imputations

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-us#4577] [General]
We estimate that lifting the SALT cap would [cost $26.2 billion in 2024](https://policyengine.org/us/policy?reform=56790&focus=policyOutput.policyBreakdown&region=enhanced_us&timePeriod=2024&baseline=2) (ECPS, static). That's based only on our state income tax model, where we assume withholding rules to be the core tax structure (no credits or deductions). We do not yet impute property taxes.

As a benchmark, PWBM recently estimated that liting the SALT cap would cost [$47 billion in 2024 and $98 billion in 2025](https://budgetmodel.wharton.upenn.edu/issues/2024/2/8/lifting-the-salt-cap-budget-effect). Something to keep in mind as we prioritize property tax imputations. @nikhilwoodruff @eqberkovich

**Comment by eqberkovich** [comment]
Should we start a DB of property tax rules? 
I don't know if one already exists, but this would be a very useful resource for the wider world.
Crowd-sourcing would help.
